src/rnnlmbin/rnnlm-train.cc

```diff
@@ -110,6 +121,11 @@ int main(int argc, char *argv[]) {
 
     po.Read(argc, argv);
 
+    core_config.backstitch_training_scale = backstitch_training_scale;
+    core_config.backstitch_training_interval = backstitch_training_interval;
+    embedding_config.backstitch_training_scale = backstitch_training_scale;
```

Comment:
why do you make them the same if you have separate options for core_config and embedding_config?

Reply:
actually i see you declared each option 3 times, but only the one defined in this .cc file take into effect.  This is very weird.

Reply:
I agree that it's weird-- I think it might be clearer if you just have two separate versions of the options that are both set from the command line, and just set them to the same values.

Reply:
Have made them two separate options in the top-level shell script.